delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ResourcesSearchParams.scala

```diff
@@ -49,6 +53,7 @@ final case class ResourcesSearchParams(
     updatedAt: TimeRange = TimeRange.Anytime,
     types: List[Type] = List.empty,
     typeOperator: TypeOperator = TypeOperator.Or,
+    fileUserMetadata: Option[FileUserMetadata] = None,
```

Comment:
`keywords` ?
In the end, it could also be extended to regular resources if users provide them

Reply:
Well it might be more than just keywords? I know that's the only field right now but soon we will have others

Reply:
Name and description will only part of full-text search